website/playground/static/index.html

```diff
@@ -128,12 +129,15 @@
   search: document.getElementById('search'),
 }, {lineNumbers: true});
 
-codeEditor.on('change', function() {
+const watch = Rx.Observable.fromEvent(codeEditor, 'change'); 
+watch.debounceTime(1000).subscribe(e => { 
   location.hash = btoa(codeEditor.getValue());
 });
 
-if (location.hash) {
+window.onhashchange = function(event) {
+  var cursorPosition = codeEditor.getCursor();
   codeEditor.setValue(atob(location.hash.slice(1)));
+  codeEditor.setCursor(cursorPosition);
 }
```

Comment:
does this fire on page load? we want to make sure that if you navigate to a playground URL that the correct code populates.

Reply:
Yes this works on load.